src/components/common/Comment.js

```diff
@@ -33,23 +109,22 @@ function Comment() {
       />
```

Comment:
above: `setNewComment` feels wrong as a name for an `onChange` handler. see why?

Reply:
We dont quite understand what you mean 🤔. It is not the name of the actual handler but rather the state setter. Could you please explain a bit more what you mean? :)